install/kubernetes/cilium/templates/cilium-agent-daemonset.yaml

```diff
@@ -554,16 +554,12 @@ spec:
           - secret:
               name: hubble-server-certs
               items:
+                - key: ca.crt
+                  path: client-ca.crt
                 - key: tls.crt
                   path: server.crt
                 - key: tls.key
                   path: server.key
               optional: true
-          - configMap:
-              name: hubble-ca-cert
```

Comment:
Is there anything blocking us from removing the `hubble-ca-cert` ConfigMap completely?

Reply:
Only my strong opinion that storing it in a `ConfigMap` is the technically correct way. I just think you should be able to grant access to the CA without granting access to a keypair.

if we were to remove it, I would suggest to do it the next release (i.e. v1.11) once we can be sure no Pod needs it during upgrade. Does that make sense?

Reply:
Yes, that does make sense! If we're planning to deprecate the ConfigMap, it's probably best to add a comment in the Helm template that it is supposed to be removed.

Reply:
As I'm not intending to remove it I don't intent to deprecated it either, but I'm open to change my mind.

Reply:
I'm not sure I follow then. What's the usecase of deploying a `ConfigMap` if it is not referenced at all?

Reply:
That's fair, we could mark it deprecated for v1.10 and see how that goes.

Reply:
@gandro I've added a comment in the `hubble-ca-configmap.yaml` template hinting that it is deprecated and will be removed in v1.11, please take another a look.